procedures/lvv-t1992.ipynb

```diff
@@ -20,239 +20,277 @@
-      "\u001b[0;32m/opt/lsst/software/stack/python/miniconda3-4.7.12/envs/lsst-scipipe-984c9f7/lib/python3.7/asyncio/tasks.py\u001b[0m in \u001b[0;36mwait_for\u001b[0;34m(fut, timeout, loop)\u001b[0m\n\u001b[1;32m    412\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    413\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mtimeout\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 414\u001b[0;31m         \u001b[0;32mreturn\u001b[0m \u001b[0;32mawait\u001b[0m \u001b[0mfut\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    415\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    416\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mtimeout\u001b[0m \u001b[0;34m<=\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
-      "\u001b[0;31mCancelledError\u001b[0m: "
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "\n",
-    "connected = await hexapod_csc.evt_connected.aget()\n",
+    "connected = await hexapod_csc.evt_connected.aget() # This is how you get an event/telemetry with await remote.type_name.aget() type being [evt,tel] and name being the name of the topic\n",
     "commandable = connected.command\n",
     "telemetry_working = connected.telemetry\n",
     "\n",
     "if not connected or not telemetry_working:\n",
     "    raise Exception(\"Hexapod not connected or telemetry not being received.\")\n",
     "\n",
+    "    #This is only for the real system. The simulater only has the AVAILABLE offline state \n",
+    "'''   \n",
     "controller_state = await hexapod_csc.evt_controllerState.aget()\n",
     "state = controller_state.controllerState\n",
     "offline_substate = controller_state.offlineSubstate\n",
-    "\n",
-    "if state == salobj.State.OFFLINE and offline_substate == Hexapod.OfflineSubstate.AVAILABLE:\n",
-    "    raise Exception(\"Controller must be changed to Available Offline Substate\")\n",
-    "\n",
+    "print(state)\n",
+    "if not state == salobj.State.OFFLINE and not offline_substate == MTHexapod.OfflineSubstate.AVAILABLE:\n",
+    "    raise Exception(\"Controller must be changed to Available Offline Substate.\")\n",
```

Comment:
I suggest the message say Offline/Available or similar -- put Available after Offline. As written it says "...Offline Substate", but Offline is not the substate, Available is the substate.